images/tidb-backup-manager/Dockerfile

```diff
@@ -2,19 +2,15 @@ FROM pingcap/tidb-enterprise-tools:latest
 ARG VERSION=v1.51.0
 ARG SHUSH_VERSION=v1.4.0
 ARG TOOLKIT_VERSION=v3.0.12
+ARG TOOLKIT_V31=v3.1.0-rc
+ARG TOOLKIT_V40=v4.0.0-rc
```

Comment:
`s/TOOLKIT/BR/g`

Reply:
But they are actually the toolkit version, BR binaries are included in the toolkit packages.
And also we may need to include other tools later in the toolkit packages.

Reply:
TOOLKIT has alot of tools I think TOOLKIT is all right.

Reply:
yes, we extract `br` from toolkit tarball but it's the implementation details. Your purpose is to specify the version of BR you wanted.

I can't understand the differences between these similar arguments.

```
ARG TOOLKIT_VERSION=v3.0.12
ARG TOOLKIT_V31=v3.1.0-rc
ARG TOOLKIT_V40=v4.0.0-rc
```

`s/TOOLKIT/TOOLKIT_BR/g` ?

Reply:
If with `TOOLKIT_BR`, what if we need to retrieve other tools, e.g. Lightning from this tarball later, as you said, `br` is implementation details, here the toolkit version is what I want.